ClassySchedule/src/main/java/scheduler/AddCourseToScheduleController.java

```diff
@@ -225,31 +274,219 @@ public LocalDateTime convertToLocalDateTimeViaInstant(Date dateToConvert) {
+        testClassNumber = validateClassNumber();
+
+        // Validation outcome for class name
+        boolean testClassName = true;
+        testClassName = validateClassName();
+
+        // Validation outcome for the start and end time
+        boolean testTime = true;
+        testTime = validateTime();
+
+        // Check all final validation outcome, if any result to false, then set final result to false
+        if(testDaysOfTheWeek == false || testDepartmentName == false || testClassNumber == false || testClassName == false || validateTime() == false) {
+            result = false;
+        }
+
+        // Return result
+        return result;
     }
 
     /**
-     * This method takes a given lecture object turn it into a string using StringBuilder to use
-     * for setting the displayCourse object strings.
-     *
-     * @param lecture a given lecture object to reference its necessary information
-     * @return a string output of the necessary lecture information
-     */
-    private String lectureToDisplayCourse(Lecture lecture) {
-        StringBuilder sb = new StringBuilder();
-        sb.append(lecture.getClassName() + "\n");
-        sb.append(lecture.getSectionNumber() + "\n");
-        sb.append(lecture.getDepartment());
-        String output;
-        output = sb.toString();
-        System.out.println(output);
-        return output;
+     * This function validates the class name. If nothing is selected, then it will prompt the user
+     * to select a class name.
+     * @return false is nothing is selected, else return true
+     */
+    private boolean validateClassName() {
+        boolean result = true;
+        // If the class name has not been selected
+        if(class_name.getSelectionModel().isEmpty()) {
+            invalidClassName.showAndWait();
+            result = false;
+        }
+        // Probably need to find a way to bind class number and class name??
+        return result;
     }
 
+    /**
+     * This function validate the class number. If nothing is selected, then it will prompt the user
+     * to select a class number.
+     * @return false is nothing is selected, else return true
+     */
+    private boolean validateClassNumber() {
+        boolean result = true;
+        // If the class name has not been selected
+        if(classNumber.getSelectionModel().isEmpty()) {
+            invalidClassNumber.showAndWait();
+            result = false;
+        }
+        // Probably need to find a way to bind class number and class name??
+        return result;
+    }
+
+    /**
+     * This function validate the department of the newly added class. If nothing is selected,
+     * then it will prompt the user to select a department.
+     * @return false is nothing is selected, else return true
+     */
+    private boolean validateDepartment() {
+        boolean result = true;
+        // If the class name has not been selected
+        if(dept_name.getSelectionModel().isEmpty()) {
```

Comment:
add a space between the if and the (
Required by the style guide. This should be done for every if you have in this PR

Reply:
Got it! Thanks!